packs/data/spell-effects.db/spell-effect-clinging-ice-critical-failure.json

```diff
@@ -0,0 +1,45 @@
+{
+    "_id": "O745ObToqOZOoCuM",
+    "img": "systems/pf2e/icons/spells/clinging-ice.webp",
+    "name": "Spell Effect: Clinging Ice (Critical Failure)",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @Compendium[pf2e.spells-srd.Clinging Ice]{Clinging Ice}.</p>\n<p>The target takes a -5-foot circumstance penalty to its Speeds until the spell ends.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": true,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "selector": "speed",
+                "type": "circumstance",
+                "value": -10
```

Comment:
Presumably this should be -5 to match the effect description?

I also think that monster speeds don't work with automatic effects, so this may not be super useful in practice.

Reply:
This is the crit fail version of the effect, which is -10

Reply:
Then the description in this effect should have the crit fail text

Reply:
Oops, fixed